run.py

```diff
@@ -133,7 +133,7 @@
                 continue
 
             # Remove line that filters DESCRIPTION with grep
-            if re.match('.*grep -v \'\\^Priority: \' DESCRIPTION.old > DESCRIPTION', line):
+            if re.match('.*grep -va* \'\\^Priority: \' DESCRIPTION.old > DESCRIPTION', line):
```

Comment:
@brendanf What is the reason for using `*` in the Python regex? I confirmed that `-va?` also works for the Python script. From the perspective of consistency (between the Python and R scripts) and explicitness (we only expect 0 or 1 `a`), I would favor `-va?`.

Reply:
@jdblischak I agree it would be better to use `?` in both cases.  I think that because I was looking at a `grep` command, the first though that came into my head was that `?` isn't supported.  Of course, in this case the `grep` command is just text in the regex, it's not actually being interpreted by `grep`.